### ISSUE TITLE
Tasks from tasks.json are not detected anymore

### ISSUE THREAD

**Issue by isometriq** [lkytal/quickTask#5]
Since the latest update to 1.11.*, the tasks from the file ./.vscode/tasks.json seem to not be detected anymore. The gulp and npm tasks from my project are appearing.

It was working prior to 1.11

Thanks for this extension, it's reaaally useful

**Comment by isometriq** [comment]
task.json contents, if that can help:
```
{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "0.1.0",
    "command": "npm",
    "isShellCommand": true,
    "showOutput": "always",
    "suppressTaskName": true,
    "_runner": "terminal",
    "tasks": [
        {
            "taskName": "install",
            "args": ["install"]
        },
        {
            "taskName": "update",
            "args": ["update"]
        },
        {
            "taskName": "test",
            "args": ["run", "test"]
        },
        {
            "taskName": "build",
            "args": ["run", "build"],
            "problemMatcher": [
                "$tsc"
            ]
        },
        {
            "taskName": "start",
            "args": ["run", "start"],
            "isBuildCommand": true,
            "isBackground": true,
            "problemMatcher": [
                "$tsc-watch"
            ]
        },
        {
            "taskName": "api",
            "command": "php ${workspaceRoot}/../tts-api/artisan serve",
            "args": [],
            "isShellCommand": true,
            "showOutput": "always"
        },
        {
            "taskName": "chrome",
            "suppressTaskName": true,
            "command": "cmd /C start chrome -incognito \"http:\/\/localhost:3000\/dist\/examples\"",
            "args": [],
            "isShellCommand": true,
            "showOutput": "never"
        }
    ]
}
```
All these tasks are working using the "Tasks: Run Task" (via F1 command line).

**Comment by lkytal** [comment]
Fixed in 2.5.1

**Comment by isometriq** [comment]
Fantastic! and thanks!
